packs/feats/dire-growth.json

```diff
@@ -33,10 +33,20 @@
         },
         "rules": [
             {
-                "key": "CreatureSize",
+                "key": "RollOption",
```

Comment:
This is an "always on" ability, so the Roll Option isn't required.

Reply:
The Dire Growth Class Feat (HotW, pg. 78) grants the Enlarge Effect as a 2-Action-Activity additionally to the normal Change Shape Activity. The current version would force you to going large every time you change into your Animal Shape.

This could also have been done by a separate effect, but since Change Shape is a Roll Option, too, I thought it's easier to use it as another Roll Option.

Reply:
@simonward - when you get a moment. Effect or Toggle? I feel the effect is probably the way to go, but I'll defer to your judgment.

Reply:
Either should be fine, since this doesn't have a specific time limit.